### DISCUSSION TITLE
issues in powershell login to azure via VSCode and progress of learning module

### DISCUSSION THREAD

**Discussion by codevshl** [github/india#612] [Students]
Hi sir/mam.
1.I'm Not able to login into my azure using powershell command in VScode. Wasted lot of time even here.
2.Progress of modules not syncing with MS Learn. Completed the modules more than two times, tried every possible way but still not synced.
Looks like we have to use all our time in solving the issues rather than utilizing it to learn properly.
 
![Screenshot (139)](https://user-images.githubusercontent.com/91673923/143271396-d5f33f49-e39b-4e03-b912-7a0622222280.png)
![image_2021-11-24_212407](https://user-images.githubusercontent.com/91673923/143271534-71146dad-039f-4e0d-a83e-bf50abcb812e.png)

**Comment by vishwakarma-shivam** [comment]
You have to add the path to **Environment Variables**.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete together.
One-stop place for daily Events/Workshops + Hackathons + Internships/Jobs + Free Scholarship, Courses, Swags & Goodies + 24/7 Live Chat and Mentors to help you with your Doubts + 24/7 Live Music & much more...

Currently, we are helping all the Future Ready Talent Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: #353

Thank you 🙏
